guides/05_custom-components/01_quick-start.md

```diff
@@ -0,0 +1,120 @@
+# Custom Components in 5 minutes
+
+Gradio 4.0 introduces Custom Components -- the ability for developers to create their own custom components and use them in Gradio apps.
+You can publish your components as Python packages so that other users can use them as well.
+Users will be able to use all of Gradio's existing functions, such as gr.Blocks, gr.Interface, API usage, themes, etc. with Custom Components.
+This guide will cover how to get started making custom components.
+
+## Installation
+
+You will need to have:
+
+* [Python 3.8+](https://www.python.org/downloads/)
+* [Node.js v16.14+](https://nodejs.dev/en/download/package-manager/)
+* [npm 9+](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm)
+* [gradio 4.0](https://pypi.org/project/gradio/)
+
+## The Workflow
+
+The Custom Components workflow consists of 4 steps: create, dev, build, and publish.
+
+1. create: creates a template for you to start developing a custom component.
+2. dev: launches a development server with a sample app & hot reloading allowing you to easily develop your custom component
+3. build: builds a python package containing to your custom component's python and javascript code -- this makes things official!
+4. publish: uploads your package to [PyPi](https://pypi.org/) and/or a sample app to [HuggingFace Spaces](https://hf.co/spaces).
+
+Each of these steps is done via the Custom Component CLI. You can invoke it with `gradio cc` or `gradio component`
+
+> 💡 Tip: Run `gradio cc --help` to get a help menu of all available commands. You can also append `--help` to any command name to bring up a help page for that command, e.g. `gradio cc create --help`.
```

Comment:
@freddyaboulton I changed the tips, now you just put "Tip: \<the text\>" on a line on its own (empty line above and below) and it will look like this: 

![Screen Shot 2023-10-29 at 10 11 35 PM](https://github.com/gradio-app/gradio/assets/9021060/e7dc5375-363a-4f32-8189-62a840665ca2)

Wdyt? I changed the existing tips to new format so they all work now.

Reply:
It looks awesome @aliabd !